app/assets/javascripts/components/course_creator/course_form.jsx

```diff
@@ -17,7 +18,11 @@ const CourseForm = (props) => {
     props.updateCourseProps({ private: isPrivate });
     props.updateCourseAction('private', isPrivate);
   };
-
+  const updateTaSupport = (e) => {
```

Comment:
This update logic looks like it's exactly the same for both checkboxes, just based on the prop key for one versus the other.  It can probably be moved to the `CourseCheckbox` component.